pages/search/search.scss

```diff
@@ -0,0 +1,19 @@
+.search-page {
+  .help-query {
+    @include scaleText( 12px, 16px, 12px, 16px);
+
+    border-radius: 10px;
+    border: 0.8px solid $black;
+    padding: 1px 8px;
+    text-transform: capitalize;
+    font-weight: 300;
+    margin-left: 4px;
+  }
+
+  .query-text {
+    @include scaleText( 12px, 16px, 16px, 24px);
+
+    display: flex;
```

Comment:
What's the need for this?

Reply:
This vertically centers the pill outline with the rest of the query-text and makes it more narrow to match design.

Reply:
Hm, ok. It feels slightly weird, but I can't put my finger on a reason so I'll let it go.

Reply:
Yeah, I know...but this is where that comes into play https://github.com/mozilla/network-pulse/pull/1070/files#diff-9b30595288e92d1997d202ae6f2ca348R17 Of course, if it was just text, there wouldn't really be a problem, but since it's text inside of a capsule I decided to do it this way primarily for aesthetic tbh ...it's a 2px diff 🤷‍♀